docs/reference/search/search-your-data/retrieval-augmented-generation.asciidoc

```diff
@@ -0,0 +1,77 @@
+[rag-elasticsearch]
+== Retrieval augmented generation
+
+.🍿 Prefer a video introduction?
+***********************
+Check out https://www.youtube.com/watch?v=OS4ZefUPAks[this short video] from the Elastic Snackable Series.
+***********************
+
+Retrieval augmented generation (RAG) is a technique where additional context is retrieved from an external datastore before prompting a language model to generate a response using the retrieved context.
+This grounds the model with in-context learning.
```

Comment:
This sentence is a little colloquial - could rephrase to clarify what you mean by "grounds"

Reply:
👍 [grounding](https://cloud.google.com/vertex-ai/generative-ai/docs/grounding/overview) is the term of art but agree shouldn't assume knowledge